.travis.yml

```diff
@@ -45,6 +45,12 @@ jobs:
         - go get github.com/myii/maintainer
         - maintainer contributor
 
+        # Update Tables of Content in the relevant `.md` files
+        - npm install markdown-toc -D
+        - markdown-toc -i CONTRIBUTING.md
+        # - markdown-toc -i README.md
```

Comment:
I've put this in here since using `README.md` is common across our repos.  That also begs the question -- should we modify the `.rst` to `.md` in this repo, too?

Reply:
If you make me answer quickly, I'd say that `md` is more common in github projects, but sincerely don't know. I'm OK with both of them (use rst mostly when I have to write pdfs parsing them with [rst2pdf](https://github.com/rst2pdf/rst2pdf)), but I'm no strongly biased for one over the other.

If you know there's a recommended one for github projects, I'd say let's use that one?

Reply:
I'm not endorsing any particular format, I have to accommodate for three (I prefer `asciidoctor` myself).  Looking at the wider picture, Markdown is the most common, making the barrier of entry lower (i.e. for contributions).  The solutions we're currently using for automation usually work best for, or are limited to, Markdown files.  Then it simply becomes an issue of _consistency_ of formats across the repo.

Reply:
There is comparison of RST and MD here: http://hyperpolyglot.org/lightweight-markup 
I have used both, have preferred rst when I want to include media/links in the markup.

Reply:
@noelmcloughlin Thanks for that.  None of us is doubting which is the _superior_ format.  This is really about which is the most _accessible_, both for other contributors as well as the automated solutions that we're introducing.  From that angle, Markdown can't be denied.  We're using it throughout GitHub and everywhere else, wherever heavy-lifting isn't required.  And the docs stored in the main repo are hardly going to be that.  If we ever get around to producing some serious documentation or even populating the formulas' wikis, the other formats come into their own.